c_src/filter.h

```diff
@@ -328,19 +328,21 @@ class EqOperator<unsigned char*>: public BinaryExpression<bool, unsigned char*>
     virtual ~EqOperator() {};
 
     virtual bool evaluate() const {
+        // [] to represent NULL w/i Riak TS, which does NOT conflict w/ <<"">>.
+        if(right_->size() == 0) {
+            return !left_->has_value();
+        }
```

Comment:
As I understand, because of the way the query compiler prepares the operand nodes for the filter, here `*left_` is an `ExpressionNode` object representing a field value as retrieved from the file, and `*right_` is the thing it is compared to, i.e., some constant or NULL. The NULLs read from file do indeed have `false == has_value()`, while those on the RHS are of type `DataType::STRING` (?) but an empty value. This should work, unless there is a constructor with `left` and `right` roles swapped (which is hard to positively verify).

(If my understanding is correct, then no action is required.)